internal/app/decision_maker.go

```diff
@@ -87,10 +87,12 @@ func (cs *currentState) makePlan(s *state) *plan {
 
 // decide makes a decision about what commands (actions) need to be executed
 // to make a release section of the desired state come true.
-func (cs *currentState) decide(r *release, n *namespace, p *plan, c *chartInfo) {
+func (cs *currentState) decide(r *release, n *namespace, p *plan, c *chartInfo, settings config) {
 	// check for presence in defined targets or groups
 	if !r.isConsideredToRun() {
-		p.addDecision("Release [ "+r.Name+" ] ignored", r.Priority, ignored)
+		if !settings.SkipIgnoredApps {
```

Comment:
nit: can you use `&&` instead of nesting?

Reply:
I don't think `and` here will do the job as we're deciding on adding or not adding the decision in two states of `SkipIgnoredApps` but the return must happen in both cases

Reply:
You're right, I missed that.